test/util.go

```diff
@@ -49,36 +62,43 @@ func AssertImageMatches(t *testing.T, masterFilename string, img image.Image) bo
 }
 
 // Tap simulates a left mouse click on the specified object.
-func Tap(obj fyne.Tappable) {
+func Tap(obj TappableCanvasObject) {
 	TapAt(obj, fyne.NewPos(1, 1))
 }
 
 // TapAt simulates a left mouse click on the passed object at a specified place within it.
-func TapAt(obj fyne.Tappable, pos fyne.Position) {
-	if focus, ok := obj.(fyne.Focusable); ok {
-		if focus != Canvas().Focused() {
-			Canvas().Focus(focus)
+func TapAt(obj TappableCanvasObject, pos fyne.Position) {
```

Comment:
Not a fan of adding new interfaces that developers won't recognise here - Can't we use Tappable?